ansible/README.md

```diff
@@ -305,24 +305,17 @@ the path to your OpenWhisk `ansible` directory contains spaces. To fix this, ple
 without spaces as there is no current fix available to this problem.
 
 #### Changing limits
-The default system throttling limits are configured in this file [./group_vars/all](./group_vars/all).
+The default system throttling limits are configured in this file [./group_vars/all](./group_vars/all) and may be changed by modifying the group_vars for your specific environment.
 ```
 limits:
-  invocationsPerMinute: "{{ limit_invocations_per_minute | default(120) }}"
-  concurrentInvocations: "{{ limit_invocations_concurrent | default(100) }}"
+  invocationsPerMinute: "{{ limit_invocations_per_minute | default(60) }}"
+  concurrentInvocations: "{{ limit_invocations_concurrent | default(30) }}"
   concurrentInvocationsSystem:  "{{ limit_invocations_concurrent_system | default(5000) }}"
   firesPerMinute: "{{ limit_fires_per_minute | default(60) }}"
   sequenceMaxLength: "{{ limit_sequence_max_length | default(50) }}"
 ```
-These values may be changed by modifying the `group_vars` for your environment. For example,
-mac users will find the limits in this file [./environments/mac/group_vars/all](./environments/mac/group_vars/all):
-```
-limit_invocations_per_minute: 60
-limit_invocations_concurrent: 30
-limit_invocations_concurrent_system: 5000
-limit_fires_per_minute: 60
-```
-- The `limit_invocations_per_minute` represents the allowed namespace action invocations per minute.
-- The `limit_invocations_concurrent` represents the maximum concurrent invocations allowed per namespace.
-- The `limit_invocations_concurrent_system` represents the maximum concurrent invocations the system will allow across all namespaces.
-- The `limit_fires_per_minute` represents the allowed namespace trigger firings per minute.
+- The `limits.invocationsPerMinute` represents the allowed namespace action invocations per minute.
+- The `limits.concurrentInvocations` represents the maximum concurrent invocations allowed per namespace.
+- The `limits.concurrentInvocationsSystem` represents the maximum concurrent invocations the system will allow across all namespaces.
+- The `limits.firesPerMinute` represents the allowed namespace trigger firings per minute.
+- The `limits.sequenceMaxLength` represents the maximum length of actions' sequence.
```

Comment:
sorry one more change:
`of actions' sequence.` -> `of a sequence action.`

Reply:
Sure!

In my personal opinion, it might be better if there's an consistent naming.

[here (last paragraph) ](https://github.com/apache/incubator-openwhisk/blob/master/docs/triggers_rules.md):
```
For example, one can create an action sequence recordLocationAndHello
```

`action sequence` -> sequence which contains of actions.
`sequence action` -> type (subtype) of an action type (which is also a sequence). Like `Cons`.

I'm not really good at English, though.